vm/validate.go

```diff
@@ -6,8 +6,7 @@ import (
 	"github.com/dlclark/regexp2"
 )
 
-// Verification helpers
-
+// VerifyExpected is a erification helpers for testing
```

Comment:
Looks like you missed a v for verification?

Reply:
fixed